data/presets/highway/footway/crossing.json

```
@@ -8,17 +8,24 @@
         "crossing_raised",
         "access"
     ],
+    "moreFields": [
+        "flashing_lights"
+    ],
     "geometry": [
         "line"
     ],
     "tags": {
         "highway": "footway",
         "footway": "crossing"
     },
+    "terms": [
+        "crosswalk",
+        "foot path crossing",
+        "pedestrian crossing"
+    ],
     "reference": {
         "key": "footway",
         "value": "crossing"
     },
-    "searchable": false,
-    "name": "Pedestrian Crossing"
+    "name": "Crossing"
```

```suggestion
    "name": "Pedestrian Crossing"
```

I would leave the name as is, because contrary to the `highway=crossing` preset, here we can and need to distinguish between Foot and Bike crossings